relay-server/src/services/project.rs

```diff
@@ -1033,6 +1048,47 @@ impl Project {
     }
 }
 
+/// Counts the nested spans inside an [`Envelope`] for each item of type
+/// transaction that didn't have spans extracted before.
```

Comment:
Technically it's the first. Practically envelopes aren't allowed to have multiple transactions.